tests/phpunit/tests/test-wpml-elementor-translatable-nodes.php

```diff
@@ -339,7 +339,7 @@ public function node_data_provider() {
 					array( 'field' => 'email_from_name', 'type' => 'Form: Email from name', 'editor_type' => 'LINE' ),
 					array( 'field' => 'success_message', 'type' => 'Form: Success message', 'editor_type' => 'LINE' ),
 					array( 'field' => 'error_message', 'type' => 'Form: Error message', 'editor_type' => 'LINE' ),
-					array( 'field' => 'required_message', 'type' => 'Form: Required message', 'editor_type' => 'LINE' ),
```

Comment:
Doesn't this also break backwards compatibility?

Shouldn't we have both `array( 'field' => 'required_message', 'type' => 'Form: Required message', 'editor_type' => 'LINE' ),` and `array( 'field' => 'required_field_message', 'type' => 'Form: Required message', 'editor_type' => 'LINE' ),`?

Reply:
@andreasciamanna this is for tests and as far as I know, the tests are being run using the latest Elementor version which should not have the old required message field. Do I understand correctly?